src/doc/reference.md

```diff
@@ -2070,6 +2070,9 @@ macro scope.
    trait of the same name. `{Self}` will be replaced with the type that is supposed
    to implement the trait but doesn't. To use this, the `on_unimplemented` feature gate
    must be enabled.
+- `must_use` - on structs and enums, will warn if this value isn't bound to a variable.
```

Comment:
Well, both work, because they both count as "use" . It's hard not to get circular here...

On Aug 17, 2016, 14:22 -0400, Jonas Schievink <notifications@github.com>, wrote:
>
> In src/doc/reference.md (https://github.com/rust-lang/rust/pull/35754#discussion_r75178315):
>
> > @@ -2070,6 +2070,9 @@ macro scope. > trait of the same name. `{Self}` will be replaced with the type that is supposed > to implement the trait but doesn't. To use this, the `on_unimplemented` feature gate > must be enabled. > +- `must_use` - on structs and enums, will warn if this value isn't bound to a variable.
>
> This has little to do with binding to variables - after all, .unwrap() makes the warning go away
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub (https://github.com/rust-lang/rust/pull/35754/files/3f54852aa06c063b5c4b08846f583ac6d57cfc82#r75178315), or mute the thread (https://github.com/notifications/unsubscribe-auth/AABsiuZJ1yZrRya9Qz8pn55F9oKkfVSvks5qg1F8gaJpZM4Jmqe9).
>
>
>